### ISSUE TITLE
[Feature] Shuffle mangled identifier alphabet

### ISSUE THREAD

**Issue by da411d** [javascript-obfuscator/javascript-obfuscator#656]
**Example code:**
```
function foo(bar, baz, bak, bos){
  return bar + baz * bak - bos;
}
```

Current output:
```
function a(b, c, d, e) {
    return b + c * d - e;
}
```

Output with shuffled alphabet:
```
function k(u, z, p, b) {
    return u + z * p - b;
}
```

**Comment by sanex3339** [comment]
Interesting. Looks like it can be easily done.

**Comment by sanex3339** [comment]
Probably, the best way to do it - is to create the new `mangled-shuffled` (that extends `mangled` generator) identifier names generator.
Because if I want to change base `mangled` generator - a huge amount of tests will fail because the order of variable names changes. But I need to know the correct order in the tests.

I can `stub` base order but here so many places where I have to `stub` it, so the best way is to create the new generator instead. 

`mangled-shuffled` sounds correct?

**Comment by sanex3339** [comment]
Merged. Will be released soon

**Comment by sanex3339** [comment]
Released as `1.5.0`

**Comment by sanex3339** [comment]
Released `1.5.2` with few fixes
